graylog2-web-interface/src/components/lookup-tables/fixtures.ts

```diff
@@ -0,0 +1,58 @@
+/*
```

Comment:
Usually the fixtures are not part of the `src` directory, they are located at `tests/fixtures`.